src/components/Container/errorhubContainer.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Container from '@mui/material/Container';
+import {Button} from '@mui/material';
+import TextField from '@mui/material/TextField';
+import {Menu1, Menu2, Menu3, Menu4}from "../Dropdown/Dropdownmenu";
+
+const ErrorhubContainer = () => {
+    return (
+        <Container sx={{width: '70%'}} className="rounded-3xl m-0 mb-0 h-[779px] shadow-md hover:shadow-xl bg-white">
+            <div className="pl-4 pr-4">
+                <div className="">
+                    <div className="text-black text-[21px] pt-8">
+                        <span>Upload your ebook to</span>
+                        <span> </span>
+                        <span className="text-blue font-semibold">ErrorHub</span>
+                    </div>
+                    <div className="pt-4">
+                        <div className="pb-2 text-black text-base capitalize">Book Title</div>
+                        <TextField
+                            sx={{ width: '100%'}}
+                            placeholder="To Kill a Mockingbird"
+                            variant="outlined"
+                            hiddenLabel
+                            disabled // Disable the input field
+                        />
+                    </div>
+                    <div className="pt-4">
+                        <div className="pb-2 text-black text-base capitalize">Author(s)</div>
```

Comment:
```suggestion
                        <div className="pb-2 text-black text-base">Author(s)</div>
```
capitalization made the "(s)" into a "(S)"